backend/src/Controllers/castVoteController.ts

```diff
@@ -78,6 +79,12 @@ async function castVoteController(req: IRequest, res: any, next: any) {
     }
 
     const savedBallot = await persistBallotToStore(inputBallot, roll, req);
+
+    if (user.email){
+        const url = req.protocol + '://' + req.get('host')
+        const receipt = Receipt(targetElection, user.email,savedBallot, url)
+        EmailService.sendEmails([receipt])
```

Comment:
looks like sendEmails is async, so this should be awaited

Reply:
Thanks